components/style/traversal.rs

```diff
@@ -773,48 +773,52 @@ where
 
             // Now that our bloom filter is set up, try the style sharing
             // cache. If we get a match we can skip the rest of the work.
-            let target = StyleSharingTarget::new(element);
-            let sharing_result = target.share_style_if_possible(context, data);
-
-            if let StyleWasShared(index, had_damage) = sharing_result {
-                context.thread_local.statistics.styles_shared += 1;
-                context.thread_local.style_sharing_candidate_cache.touch(index);
-                return had_damage;
-            }
-
-            context.thread_local.statistics.elements_matched += 1;
 
+            // This is only relevant for animations as of right now.
             important_rules_changed = true;
```

Comment:
Well, we never cache styles with animations, so it was probably OK (except for web animations maybe?)